paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -151,7 +153,7 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentSheetResult>() {
             googlePayButton.isEnabled = enabled
         }
 
-        viewModel.selection.observe(this) {
+        viewModel.selection.filterNotNull().launchAndCollectIn(this) {
```

Comment:
This is different from before, where we wouldn’t filter out `null`. Why the change?

Reply:
@jaynewstrom-stripe do you remember why selection needed to be filtered?

Reply:
Looks like it's due to this change. https://github.com/stripe/stripe-android/pull/5999/files#diff-bbc3f2e9350bfc93939031fff087aa439e01391d7fc521f13b9fd605d05da615R159

LiveData lies. It allows you to declare something non null, but in reality, the `.value` is null if it's uninitialized.

The way around this is likely to make this not a `StateFlow`. I'm not sure which tradeoff is better.